README.md

```diff
@@ -59,7 +59,7 @@ You can sort/filter/download the data from [Airtable](https://airtable.com/shrXv
 
 - [Daanfe Inc Pvt. Ltd.](http://www.daanfe.com/) | Sankhamul Road, Kathmandu | Wings For Ideas
 - [Danfe Solution Pvt. Ltd.](http://danfesolution.com/) | Gyaneshwor, Kathmandu | IT Company focusing on Web Development, Mobile Application and Desktop Application
-- [Danphe Infotech Pvt. Ltd.](http://www.danpheinfotech.com/) | Kupondole, Lalitpur | We specialize in Software and Web Development
+- [Danphe Software Labs Pvt. Ltd.](http://www.danpheinfotech.com/) | Anamnagar, Kathmandu  | We specialize in Software and Web Development
```

Comment:
Thank you @cdrrazan for the update. The website is unfortunately not working and it is one of the criteria that needs to be fulfilled to be in this list.

Reply:
On second thought, I might need to add a bot to check the URLs so that companies with non-working URLs can be taken off the list.

Reply:
http://danpheit.com/

I think the URL should be this one?